test/server/wtr-config.test.js

```diff
@@ -0,0 +1,120 @@
+import { createConfig, getBrowsers, WTRConfig } from '../../src/server/wtr-config.js';
+import { createConfig as createConfigPublic, getBrowsers as getBrowsersPublic } from '../../src/index.js';
+import { expect } from 'chai';
+import sinon from 'sinon/pkg/sinon-esm.js';
+
+describe('createConfig()', () => {
+
+	it('should be exported from index', () => {
+		expect(createConfig).to.equal(createConfigPublic);
+	});
+
+	it('calls WTRConfig.create', () => {
+		const createStub = sinon.stub(WTRConfig.prototype, 'create').callsFake(num => num / 2);
+		const res = createConfig(10);
+
+		sinon.assert.calledOnce(createStub);
+		sinon.assert.calledWith(createStub, 10);
+		expect(res).to.equal(5);
+
+		createStub.restore();
```

Comment:
I'd suggest moving the `restore` call into a common `afterEach`. Otherwise if one of these `assert`s fails, `restore` won't get called and the rest of your tests will start behaving weirdly, possibly making it more difficult to diagnose what actually went wrong.